src/SprykerEco/Zed/Payone/Communication/Plugin/Log/PaymentLogReceiverPlugin.php

```diff
@@ -28,12 +30,13 @@ class PaymentLogReceiverPlugin extends AbstractPlugin implements PaymentLogRecei
      *
      * @return \Generated\Shared\Transfer\PayonePaymentLogCollectionTransfer
      */
-    public function getPaymentLogs(ObjectCollection $orders)
+    public function getPaymentLogs(ObjectCollection $orders): PayonePaymentLogCollectionTransfer
```

Comment:
It's a part of public API, please remove added type hints.

Reply:
Fixed.